corehq/apps/userreports/management/commands/queue_indicators.py

```diff
@@ -8,9 +8,10 @@
 
 class Command(BaseCommand):
 
-    def handle(self, **options):
-        indicators_by_domain_doc_type = defaultdict(list)
-        for indicator in AsyncIndicator.objects.all():
-            indicators_by_domain_doc_type[(indicator.domain, indicator.doc_type)].append(indicator)
-        for k, indicators in indicators_by_domain_doc_type.items():
-            _queue_indicators(indicators)
+    def add_arguments(self, parser):
+        parser.add_argument('data_source_ids')
+        parser.add_argument('--agg-queue', dest='use_agg_queue', action='store_true')
+
+    def handle(self, data_source_ids, use_agg_queue, **options):
+        indicators = AsyncIndicator.objects.filter(indicator_config_ids__contains=[data_source_ids])
```

Comment:
why we don't need to group by doc_type anymore?
`data_source_ids` is mandatory and in case we have multiple passed, there will be multiple doc_types fetched.

Reply:
Oh yeah, this command definitely assumes you only pass 1 `data_source_id`, i can update the help text to make that clear and remove the `s`. If you need to queue multiple doc types that will take multiple commands

Reply:
cool, sounds good